x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameRestTestCase.java

```diff
@@ -211,7 +211,8 @@ protected void createPivotReviewsTransform(String transformId, String dataFrameI
                 + "     \"avg_rating\": {"
                 + "       \"avg\": {"
                 + "         \"field\": \"stars\""
-                + " } } } }"
+                + " } } } },"
+                + "\"frequency\":\"1s\""
```

Comment:
Default frequency is low (meaning more time between triggered actions) by default. Making it `1s` for our tests so that things can progress more quickly.